src/tools/make-headers.r

```diff
@@ -81,11 +82,45 @@ emit-proto: proc [proto] [
             emit-rlib ["extern " proto "; // " the-file]
         ][
             emit-out ["extern " proto "; // " the-file]
+            either "REBTYPE" = proto-parser/proto.id [
+               emit-fsymb ["    SYM_FUNC(" proto-parser/proto.arg.1 "), // " the-file]
```

Comment:
This line needs to be like (manually expand the macro REBTYPE):
               emit-fsymb ["    SYM_FUNC(T_" proto-parser/proto.arg.1 "), // " the-file]